src/services/DownloadService.php

```diff
@@ -22,14 +23,21 @@ class DownloadService implements DownloadServiceInterface
      */
     private $fileSystem;
 
+    /**
+     * @var DirectoryServiceInterface
+     */
+    private $directoryService;
```

Comment:
Add the variable type instead of a comment like you did on `src/services/DirectoryService.php`. You can also do the same on the `$fileSystem` variable.